src/pages/dapp/index.tsx

```diff
@@ -313,7 +358,8 @@ function ReviewDialog(props) {
           </svg>
         </button>
         <Column>
-          Have you opened or downloaded dapp before posting review?
+          <p>You have not open or download dapp before posting review.</p>
```

Comment:
you have not `opened`

(or) 

You did not visit or download the dapp before to post a review